rollup.config.mjs

```diff
@@ -30,7 +29,7 @@ export default [
         input: 'src/index.ts',
         output: {
             banner,
-            file: pkg.main,
+            dir: pkg.main,
```

Comment:
Was getting a `[!] RollupError: Invalid value for option "output.file" - when building multiple chunks, the "output.dir" option must be used, not "output.file". To inline dynamic imports, set the "inlineDynamicImports" option.` error at build time without this change.

Reply:
If we're changing this, we either need to:

1. Change the package.json so its not specifying filenames, and is instead specifying folder names. Right now its generating folders with filenames, you can see them in `./lib`. 
2. Switch instead to using `inlineDynamicImports` instead, which will output a file and we can revert these changes. This will bundle it all into one file.

Unsure of which is better TBH. Worth some research?

Reply:
My understanding is that using `inlineDynamicImports` will just bundle everything the way it was bundled before this PR so it's probably fine 🤔

Reply:
Changed the rollup configs to use `inlineDynamicImports` instead 👍